regression/cbmc/integer-assignments1/main.c

```diff
@@ -0,0 +1,9 @@
+int main()
+{
+  __CPROVER_integer a, b;
+  a=0;
+  b=a;
+  b++;
+  b*=100;
+  __CPROVER_assert(0, "");
```

Comment:
What about a test that tests the property that distinguishes integers from bitvectors:
```
__CPROVER_integer a, b;
b = a + 1;
__CPROVER_assert(b > a, "");
```